ha/resource/resource_agent.py

```diff
@@ -192,14 +192,8 @@ def stop(self):
         Log.debug(f"In stop for {filename}")
         if os.path.exists(const.HA_INIT_DIR + filename):
             os.remove(const.HA_INIT_DIR + filename)
-        while True:
-            if self.monitor() == const.OCF_NOT_RUNNING:
-                time.sleep(2)
-                break
-        if self.monitor(state=const.STATE_STOP) == Action.RESTART:
-            Log.debug(f"Restarting {filename} resource")
-            time.sleep(2)
-            return const.OCF_SUCCESS
+            Log.debug(f"Stopping {filename} resource")
+        Log.debug(f"Stopped {filename} resource return success")
```

Comment:
"Stopped {filename} resource successfully" maybe?

Reply:
Thanks @knekrasov 
yes look like 
Log.debug(f"Stopping {filename} resource")
is not useful log and need to improve msg on next log


as patch is merged i will create github issue and take it as improvement in next check-in

Reply:
https://github.com/Seagate/cortx-ha/issues/193